### DISCUSSION TITLE
Error while performing make check

### DISCUSSION THREAD

**Discussion by pvsai-sushant** [LLNL/Silo#269] [Q&A]
Hi,
   As you have written in the Install file regarding the compilation of Silo 4.11RC to Linux, when I run the command of make check, it displays the following error: 
ioperf_pdb.c(56): catastrophic error: cannot open source file "lite_pdb.h"
Can you please tell me where I am going wrong and help me out in the successful installation of the SILO package?
Thanks & Regards,
Sushant
(P.S.: The steps I followed are unzipping the zip file using the unzip command; then running "./configure" command; then the "make" command and finally "make check" command)

**Comment by MingYang-FEMAI** [comment]
I have encountered precisely the same trouble, and it took me a few hours to find the "_lite_pdb.h_" header file in my computer and LLNL organization, but unfortunately, nothing was found. 
I guess this trouble will be solved if the header file "lite_pdb.h" was plugged into the "_/silo-4.x.x/tests/_" folder.

**Reply by pvsai-sushant** [reply]
So you are saying to try creating an empty header file with the same name in the required folder?
- If that's the case, then the surprise is that it is still showing a similar (following) error:
ioperf_pdb.c(64): error: identifier "PDBfile" is undefined
  static PDBfile *pdbfile;
         ^
ioperf_pdb.c(68): warning #556: a value of type "int" cannot be assigned to an entity of type "<error-type> *"
      pdbfile = lite_PD_create(filename);
              ^
compilation aborted for ioperf_pdb.c (code 2)
make[1]: *** [ioperf_pdb_la-ioperf_pdb.lo] Error 1
make[1]: Leaving directory `/home/vpaturu2/thesis/tp/SILO/tests'
make: *** [check-am] Error 2

The above "ioperf_pdb.c" file is the one which calls the header function - "lite_pdb.h"

**Reply by MingYang-FEMAI** [reply]
Obviously, an empty header file will not work.  What I meant was that someone who had correctly installed this repo could offer you a correct header file.

**Comment by SimonFishhh** [comment]
I'm also having the same issue, did you fix this？
I would be very grateful to have your help

**Comment by markcmiller86** [comment]
Am away for week of vacation. Will look at this asap after November 28th.

**Reply by SimonFishhh** [reply]
Can I get the previous versions, which were available for download on this site( http://software.llnl.gov/Silo/ )yesterday?

**Reply by markcmiller86** [reply]
Hmmm....those older releases should still be available at the old URLs. It looks like our Web team here may have inadvertently hidden those pages. I will make an inquiry now and investigate further when I return from vacation.

**Reply by markcmiller86** [reply]
I checked in spack and it looks like you can find the raw URLs to `wget` or `curl` those files here...

https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/silo/package.py

**Reply by SimonFishhh** [reply]
Oh~ I got it, thank you so much

**Comment by Wewannado** [comment]
I ran into this same problem, and solved with:
./config/mklite ./src/pdb/pdb.h  ./src/pdb/lite_pdb.h
And also
./config/mklite ./src/score/score.h  ./src/scpre/lite_score.h
